receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -33,12 +34,14 @@ import (
 )
 
 const (
-	traceIDKey = "trace_id"
-	spanIDKey  = "span_id"
+	traceIDKey           = "trace_id"
+	spanIDKey            = "span_id"
+	PrometheusUntypedKey = "promethues_untyped_metric"
```

Comment:
@ridwanmsharif sorry but I just noticed this typo: `promethues_untyped_metric` -> `prometheus_untyped_metric`

Reply:
good catch 🤦

Reply:
Sent over https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/172